<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Paradigm_Advance</class>
 <widget class="QDialog" name="Dialog_Paradigm_Advance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>691</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>  Num of volume: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="in_NumVolume"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_TaskName">
          <property name="text">
           <string>         Task Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="in_TaskName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="add_Task_Btn">
          <property name="text">
           <string>Add Task</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="del_Task_Btn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>                 TR(s):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="in_TR"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_ConditionName">
          <property name="text">
           <string>   Condition Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="in_ConditionName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_CondiitionVector">
          <property name="text">
           <string>        Onset (TRs):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="in_Onset"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>   Durations (TRs):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="in_duration"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="add_Condition_Btn">
          <property name="text">
           <string>Add Condition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="del_Condition_Btn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_Extern_Btn">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Wave Form:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>BoxCar</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Convolution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>HRF</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>ContrastVector    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="in_ContrastVector"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="valid_vector_Btn">
        <property name="text">
         <string>Validate Vector</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="done_advance_Btn">
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_11">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Task List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="task_List_Widget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Condition List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="condition_list_Widget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Design Matrix</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="designMatrix_Graph"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>670</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QPushButton" name="view_Design_Btn">
      <property name="text">
       <string>View DesignMatrix</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="select_Task_Btn">
      <property name="text">
       <string>     Select Task     </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
